packages/engine/Source/Scene/GlobeSurfaceTileProvider.js

```diff
@@ -532,10 +532,13 @@ GlobeSurfaceTileProvider.prototype.endUpdate = function (frameState) {
       const tile = tilesToRender[tileIndex];
       const tileBoundingRegion = tile.data.tileBoundingRegion;
       addDrawCommandsForTile(this, tile, frameState);
-      frameState.minimumTerrainHeight = Math.min(
-        frameState.minimumTerrainHeight,
-        tileBoundingRegion.minimumHeight
-      );
+      frameState.minimumTerrainHeight =
+        tileIndex === 0
```

Comment:
Can you explain why the `tileIndex === 0` condition is here?

Reply:
Makes sense! Thanks for the explanation. In that case, is it even necessary to use  `ApproximateTerrainHeights._defaultMinTerrainHeight`?

Reply:
If there are no tiles in the view, then `frameState.minimumTerrainHeight` is not updated and its value is whatever was set as default value. It is `0.0` in the current implementation, while the goal of this update is to set it to `ApproximateTerrainHeights._defaultMinTerrainHeight`